tests/end_to_end/test_brute_force/test_brute_force_rdp/test_brute_force_rdp.py

```diff
@@ -26,7 +26,7 @@
 @pytest.mark.parametrize('metadata', configuration_metadata, ids=cases_ids)
 def test_brute_force(configure_environment, metadata, get_dashboard_credentials, generate_events, clean_alerts_index):
```

Comment:
```suggestion
def test_brute_force_rdp(configure_environment, metadata, get_dashboard_credentials, generate_events, clean_alerts_index):
```

Reply:
Done in 123ef4c40e7ce10616c8601a498cd48decfe62e7